ferrocene/doc/safety-manual/src/constraints.rst

```diff
@@ -47,12 +47,6 @@ Constraints
      - The user shall not perform source modification when the build of the
        safety-related software is in progress.
      - :id:`AVD_PARALLEL_BUILD_006`
-   * - .. id:: CSTR_0070_TEST
```

Comment:
Why was this removed?

Reply:
The first bullet point talked about `Remove mention of ISO 26262`, I assumed it meant this?

Reply:
Oh, remove the mention in the same document ("A CLI option or one of its arguments may be outside the scope of the Ferrocene |iso_ref| qualification"), just having "..outside the scope of the Ferrocene qualification").